WickedEngine/wiGraphicsDevice_Vulkan.cpp

```diff
@@ -5761,6 +5765,8 @@ using namespace vulkan_internal;
 				view_desc.format = VK_FORMAT_D32_SFLOAT_S8_UINT;
 				view_desc.subresourceRange.aspectMask = VK_IMAGE_ASPECT_DEPTH_BIT;
 				break;
+			default:
+				assert(0);
```

Comment:
why not break?

Reply:
because the assert will just exit the program anyway, right?
oh! But only in debug mode, otherwise they are compiled out. It doesn't matter because it's the last statement of the switch, but for style consistency I could add it back.

Reply:
yes, always have break; in switches pls (unless it always returns)
The assert can be ignored even in debug mode on Windows.